jupyterlab_comments/jupyterlab_comments/__init__.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 from notebook.utils import url_path_join
-from jupyterlab_comments.handlers import DetachedCommentsHandler, ReviewCommentsHandler
+from jupyterlab_comments.handlers import DetachedCommentsHandler, ReviewCommentsHandler, RefreshIntervalHandler, PullFromRemoteRepoHandler
```

Comment:
nit: Import grouping is "recommended" but not enforced:

```
from jupyterlab_comments.handlers import DetachedCommentsHandler 
from jupyterlab_comments.handlers import ReviewCommentsHandler 
from jupyterlab_comments.handlers import RefreshIntervalHandler
from jupyterlab_comments.handlers import PullFromRemoteRepoHandler
```

Reply:
This change will be incorporated in the next PR.